src/zone.rs

```diff
@@ -35,6 +35,33 @@ impl<'a> Endpoint<Zone> for ZoneDetails<'a> {
     }
 }
 
+
+/// Zone Details
```

Comment:
We try to keep these doc comments consistent with the official API documentation endpoint titles. This particular one should be: `/// Purge Files by Cache-Tags or Host`